.github/workflows/build-and-upload-binaries-after-release.yaml

```
@@ -1,27 +1,37 @@
 name: Build and upload binaries after release
 
 on:
+  pull_request:
   release:
     types: [created]
 
 jobs:
   publish:
-    name: Build and upload for ${{ matrix.os }}
+    name: Build and upload for ${{ matrix.platform }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         include:
-          - os: ubuntu-latest
+          - platform: Linux x86_64
+            os: ubuntu-latest
             artifact-path: compiler/cpp/cmake-build/bin
             asset-name: thrift-${{ github.ref_name }}-linux-x86_64.tar.gz
             install-deps: sudo apt install -yq automake bison flex cmake
             build-command: make
-          - os: macos-latest
+          - platform: Macos ARM64
+            os: macos-latest
             artifact-path: compiler/cpp/cmake-build/bin/Debug
             asset-name: thrift-${{ github.ref_name }}-darwin-arm64.tar.gz
             install-deps: brew install automake bison flex cmake && echo "PATH=$(brew --prefix bison)/bin:$PATH" >> $GITHUB_ENV
             build-command: cmake --build .
             cmake-parameters: -G "Xcode"
+          - platform: Linux ARM64
+            os: ubuntu-latest
+            artifact-path: compiler/cpp/cmake-build/bin
+            asset-name: thrift-${{ github.ref_name }}-linux-aarch64.tar.gz
+            install-deps: sudo apt install -yq automake bison flex cmake g++-aarch64-linux-gnu
+            cmake-parameters: -DCMAKE_TOOLCHAIN_FILE="../../../build/cmake/Linux-ARM64.cmake"
```

Nit. Можно ещё как вариант:
```suggestion
            cmake-parameters: -DCMAKE_TOOLCHAIN_FILE="${{ github.workspace }}/build/cmake/Linux-ARM64.cmake"
```

Чтобы точки не считать.

> https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

---

Фиг там(

---

Шок. (

---

Исправил